model/src/main/kotlin/OrtResult.kt

```diff
@@ -292,7 +292,7 @@ data class OrtResult(
      */
     @JsonIgnore
     fun getOpenIssues(minSeverity: Severity = Severity.WARNING) =
-        getIssues(omitExcluded = true, omitResolved = true, minSeverity = minSeverity).values.flatten()
+        getIssues(omitExcluded = true, omitResolved = true, minSeverity = minSeverity).values.flatten().distinct()
```

Comment:
Commit message: Maybe say "The same issue can occur for different identifiers, ..."